projects/mysql-cluster/README.md

```diff
@@ -0,0 +1,222 @@
+<!--forhugo
++++
+title="MySQL Replication -  Configuration and Troubleshooting"
++++
+forhugo-->
+# MySQL Replication - Configuration and Troubleshooting
+
+This project is designed to introduce you to MySQL database setup, schema creation, replication, and failover using Amazon EC2 instances.
+
+You will set up a primary MySQL server, test it, add a secondary server for replication, and demonstrate replication and failover processes. Most steps are designed to succeed, but some will require some troubleshooting and problem-solving.
+
+## Learning Objectives
+- Install MySQL server
+- Configure MySQL as a cluster
+- Troubleshoot the cluster configuration
+- Fail-over the cluster
+
+Timebox: 5 days
+
+## Project
+
+There are different ways to configure MySQL replication. In this exercise, you will be configuring our servers for primary-replica (or master-slave) replication.
+
+In this type of replication, the primary server (or the master) takes all the writes and they are automatically replicated onto the replica server (or the slave). This technique is widely used to increase the scalability of the database for read-intensive operations (which is extremely common for the web). In the primary-replica setup, the replica (or replicas) would normally be used for reads and primary for writes only. Even though it's technically possible to use the primary for the reads and the writes, it is impossible to write directly to the replica.
```

Comment:
This talks about the scaling properties of replication, but not the resiliency properties - given the exercise involves failing over, it may be worth introducing the idea here too?

Reply:
Done